openapi.yaml

```diff
@@ -249,6 +249,17 @@ paths:
           schema:
```

Comment:
We should probably change the response code from `200` to `201` here since we've actually created a new object.